src/components/NoteNameEditor/NoteNameEditorComponent.js

```diff
@@ -16,13 +16,15 @@ export class NoteNameEditorComponent extends Component {
   debouncedOnChange = _.debounce(name => this.props.onChange(name), 1500);
 
   render() {
-    const { onDeleteClick, onUploadFileClick } = this.props;
+    const { onChangeParentClick, onDeleteClick, onUploadFileClick, changeNoteParentMode } = this.props;
 
     return (
       <StyledNoteNameEditor>
         <HeaderComponent
+          onChangeParentClick={onChangeParentClick}
           onDeleteClick={onDeleteClick}
           onUploadFileClick={onUploadFileClick}
+          changeNoteParentMode={changeNoteParentMode}
```

Comment:
It's hard to understand if `changeNoteParentMode` is a function or a string or whatever.

If it's a boolean, let's start the variable name with `is`, e.g. `isChangeParentModeActive`.